include/zephyr/drivers/gnss.h

```diff
@@ -169,7 +169,7 @@ struct gnss_time {
 	uint8_t hour;
 	/** Minute [0, 59] */
 	uint8_t minute;
-	/** Millisecond [0, 59999] */
+	/** Millisecond [0, 60999] */
 	uint16_t millisecond;
```

Comment:
What is the reasoning behind this field combining seconds and milliseconds? It seems out of place compared to everything else getting its own field.

This also drops a substantial amount of precision. GNSS modems report time accuracy in nanoseconds, and the RTC resolution of most systems will be ~30.5 microseconds from the 32768Hz LF XTAL.

Reply:
It just seems like an odd line to draw, especially when the struct is 7 bytes as is, so you're going to lose a byte to padding regardless.

Reply:
Do you want this member to be altered as part of this PR? Its not exactly related to the topic of ambiguity :)

Reply:
No, it was just convenient to ask here